common/scripted_effects/hfe_effect_pirate.txt

```diff
@@ -221,7 +221,21 @@ hfe_effect_pirate_init_colony = {
 		# log = "pirate created new base [this.GetName] in [solar_system.GetName] system, owner: [owner.GetName]"
 		# }
 		hfe_effect_create_starbase_starport_from_planet = yes
-		hfe_effect_starbase_add_shipyard_0 = yes
+		starbase = {
```

Comment:
First, indexes starts from zero.
Second, im disagree.
Pirate ai should be able to bauild everything by itself.

Reply:
That caused serious bugs. I showed them to you in the chat room, remember?

Reply:
It work on my branch

Reply:
I showed you the actual bugs and the actual screenshots of those bugs. I had screenshots. That settles the matter.

Reply:
It happened then you destroyed pirate country type

Reply:
Or whatever, work on my branch